src/diff/index.js

```diff
@@ -166,7 +166,14 @@ export function diff(parentDom, newVNode, oldVNode, context, isSvg, excessDomChi
 	return newVNode._dom;
 }
 
+/**
+ * @param {Array<import('../internal').Component>} commitQueue List of components
+ * which have callbacks to invoke in commitRoot
+ * @param {import('../internal').VNode} root
+ */
 export function commitRoot(commitQueue, root) {
+	if (options._commit) options._commit(root, commitQueue);
```

Comment:
If we need a post commit option, we should probably call it `_committed` following our existing option naming pattern

Reply:
I don't know off the top of my head, but this is arguable the one option hook where most of the devtools code is based upon. Looking at the changes here it should be fine. Just posting to make a note to myself here.

Reply:
Oh true... Hmm one workaround would be to delay the devtools code after a `Promise.then` callback